File: src/main/java/com/haozileung/web/domain/Role.java
package com.haozileung.web.domain;

/**
 * Created by haozi on 16-3-7.
 */
public class Role {

    private Long roleId;

    private String roleName;
    private Status status;

    public Status getStatus() {
        return status;
    }

    public void setStatus(Status status) {
        this.status = status;
    }

    public Long getRoleId() {
        return roleId;
    }

    public void setRoleId(Long roleId) {
        this.roleId = roleId;
    }

    public String getRoleName() {
        return roleName;
    }

    public void setRoleName(String roleName) {
        this.roleName = roleName;
    }
}

File: src/main/kotlin/com/haozileung/web/domain/permission/Role.kt
/*
 * Powered By [rapid-framework]
 * Web Site: http://www.rapid-framework.org.cn
 * Google Code: http://code.google.com/p/rapid-framework/
 * Since 2008 - 2016
 */

package com.haozileung.web.domain.permission

import java.io.Serializable

/**
 * @author Haozi
 * *
 * @version 1.0
 * *
 * @since 1.0
 */

data class Role(var roleId: Long? = null,
                var name: String? = null,
                var remarks: String? = null,
                var status: Int? = null) : Serializable {
}
